packages/gst_plugins_bad.rb

```diff
@@ -0,0 +1,67 @@
+require 'package'
+
+class Gst_plugins_bad < Package
+  description 'Multimedia graph framework - bad plugins'
+  homepage 'https://gstreamer.freedesktop.org/'
+  @_ver = '1.18.3'
+  version @_ver
+  compatibikity 'all'
```

Comment:
Misspelled compatibility?

Reply:
Yup. This explains my poor second-place finish in that one elementary school spelling bee.